.drone.yml

```diff
@@ -0,0 +1,13 @@
+kind: pipeline
+name: default
+
+steps:
+- name: build
+  image: truedoctor/rust-wasm:lts
+  commands:
+  - nix-shell && cargo make all -v # build all projects
```

Comment:
Without taking a closer look, you probably want to do something like `nix-shell --run 'build cmds'` since `nix-shell` is interactive and will wait for user-input forever in this case.

Reply:
For further reference, please take a look at the `nix-shell(1)`-manpage or the [manual](https://nixos.org/nix/manual/#sec-nix-shell).

Reply:
I need help :sob:
```
[cargo-make] INFO - cargo make 0.23.0
[cargo-make] INFO - Build File: Makefile.toml
[cargo-make] INFO - Task: all
[cargo-make] INFO - Profile: development
[cargo-make] INFO - Running Task: init
[cargo-make] INFO - Running Task: atomify
[cargo-make] INFO - Execute Command: "cargo" "make" "exec-debug" "--" "build" "-p" "atomify"
[cargo-make] INFO - cargo make 0.23.0
[cargo-make] INFO - Build File: Makefile.toml
[cargo-make] INFO - Task: exec-debug
[cargo-make] INFO - Profile: development
[cargo-make] INFO - Running Task: init
[cargo-make] INFO - Running Task: exec-debug
[cargo-make] ERROR - Error while executing command, exit code: 1
[cargo-make] WARN - Build Failed.
[cargo-make] ERROR - Error while executing command, exit code: 1
[cargo-make] WARN - Build Failed.
```

Reply:
btw. I'm pushing so many commits to trigger the build system…

Reply:
you can look at the ci output directly: https://ci.kobert.dev/

Reply:
> btw. I'm pushing so many commits to trigger the build system…

I know the struggle :p
Just hoped my comments might be helpful by resolving the issue :)

> I need help

Is this reproducible by simply running `nix-shell --run 'cargo make all'` on your branch?

Reply:
yes it is reproducible had some trouble to get it to accept the repository move, but should be fine now

Reply:
Build is still failing

Reply:
wuhu, seems to work now

Reply:
should we move all dependencies to the docker image or leave them as shell requirements?

Reply:
> should we move all dependencies to the docker image or leave them as shell requirements?

IMHO no: I still wanna use the nix-shell for local development :)

Reply:
Thats the current implementation, the docker just contains cached versions and the shell is still used during ci

Reply:
The current setup seems quite straight forward and works so far, only cargo make format has some issues. #41
Any ideas for more things to do with the ci?
otherwise I'll consider it done for now and start implementing features

Reply:
That's fine :) Will hopefully have more time for this next weekend.